src/configure/configurers/provisioningConfigurer.ts

```diff
@@ -185,15 +192,14 @@ export class ProvisioningConfigurer implements IProvisioningConfigurer {
 
     public async getPathToFile(fileName: string, directory: string) {
```

Comment:
make this function private

Reply:
And put all public functions at top and all private ones at bottom

Reply:
done